aqueductcore/backend/routers/files.py

```diff
@@ -202,3 +207,93 @@ async def upload_experiment_file(
         ) from error
 
     return JSONResponse({"result": f"Successfuly uploaded {file_name}"})
+
+
+# pylint: disable=too-many-return-statements,too-many-branches
+@router.post("/{experiment_uuid}/delete_files")
+async def remove_experiment_files(
+    request: Request,
+    experiment_uuid: UUID,
+    context: Annotated[ServerContext, Depends(context_dependency)],
+) -> JSONResponse:
+    """Router for deleting file from an experiment"""
+
+    request_body = await request.json()
+    file_list = request_body.get('file_list')
+    if not file_list:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail="File list missing from request body",
+        )
+    file_list = list(set(file_list))
+
+    if file_list is None:
+        raise HTTPException(
+            status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+            detail="File list is missing from the request body."
```

Comment:
I was wondering maybe we should have a different error message compared to `400`
```suggestion
            detail="File list can not be empty"
```